src/saving_tools.jl

```diff
@@ -103,9 +118,15 @@ function tag!(d::Dict{K, T}, gitpath = projectdir(), source = nothing) where {K,
     end
     if String <: T
         d[K("commit")] = c
+        if patch!=""
+            d[K("gitpatch")] = patch
+        end
     else
         d = Dict{K, promote_type(T, String)}(d)
         d[K("commit")] = c
+        if patch!=""
+            d[K("gitpatch")] = patch
```

Comment:
Why are you naming this `gitpatch` instead of `gitdiff`? It does directly contain the output of `gitdiff`, correct?

Reply:
Well, the diff(erencing) produces a patch.  But I'm easy.

Reply:
Alright, let's keep it patch then. I'm also easy on this.,

Reply:
There is the other inconsistency: "gitpatch" vs "commit" but I felt that "patch" alone was a bit confusing.  Maybe change "commit" -> "gitcommit"?

Reply:
hm, good point. Although breaking I think this is a good improvement (and the next release has one other breaking change anyways so let's go with it!)